app-misc/jigdo/patches/jigdo-0.7.3.patchset

```diff
@@ -0,0 +1,58 @@
+From 3226f395702e65af17db4c06348e68d088515ce7 Mon Sep 17 00:00:00 2001
+From: nys <33534144+nysnatuss@users.noreply.github.com>
+Date: Sun, 31 Dec 2017 00:25:20 +0000
+Subject: patch jigdo
+
+
+diff --git a/src/compat.hh b/src/compat.hh
+index 3a7ac9a..17c0887 100644
+--- a/src/compat.hh
++++ b/src/compat.hh
+@@ -20,6 +20,7 @@
+ 
+ #include <string>
+ #include <stdio.h>
++#include <stdlib.h>
```

Comment:
the `.hh` suffix indicate that this is an C++ header, so you may want to include `cstdlib` instead

```diff
-+#include <stdlib.h>
++#include <cstdlib>